### PR TITLE
Adding more S3 clients examples to the docs

### PR DESCRIPTION
**Pull request by andrea-manzi**
# Summary

---

**Related issue :**

### CODE REVIEW

**Comment by enolfc** [comment]
should we add these clients to the notebooks?

**Comment by sebastian-luna-valero** [comment]
Very good idea, I would actually say `cvmfs` for 
* https://docs.min.io/docs/minio-client-quickstart-guide

Conda for:
* https://anaconda.org/conda-forge/awscli
* https://anaconda.org/conda-forge/davix
* https://anaconda.org/conda-forge/s3fs
* https://anaconda.org/conda-forge/minio

File: content/en/users/data/storage/object-storage/_index.md [hunk 1]
```diff
@@ -342,7 +339,50 @@ to access object storage offered by other providers.
...
+First of all the configuration of the access and secret keys need to be done:
+
+```shell
+$ aws configure
+```
+
+then it offers many commands to list, create buckets, objects, e.g.:
+
+```shell
+$ aws  s3 ls --endpoint-url  https://object-store.cloud.muni.cz  s3://test-egi-2
```
Comment (sebastian-luna-valero) [inline content/en/users/data/storage/object-storage/_index.md:358]:
```suggestion
$ aws s3 ls --no-sign-request --endpoint-url https://object-store.cloud.muni.cz s3://test-egi-public
```

**Comment by andrea-manzi** [comment]
the linter is complaining on a GGUS page, cause `api` is not uppercase in this page

https://docs.egi.eu/internal/configuration-database/service-types/#operational-components-middleware-agnostic

 see `argo.api`

@gwarf what do you suggest?

File: content/en/users/data/storage/object-storage/_index.md [hunk 2]
```diff
@@ -342,7 +339,50 @@ to access object storage offered by other providers.
...
+The [MinIO CLI](https://docs.min.io/docs/minio-client-quickstart-guide)
+supports filesystems and Amazon S3 compatible cloud storage services.
+
+It offers a modern alternative to UNIX commands like ls, cat, e.g.:
+
+```shell
+# key and secret are not mandatory in case of public buckets
+$ ./mc alias set cesnet https://object-store.cloud.muni.cz
+
+$ ./mc ls cesnet/test-egi-2
```
Comment (sebastian-luna-valero) [inline content/en/users/data/storage/object-storage/_index.md:376]:
```suggestion
$ ./mc ls cesnet/test-egi-public
```

File: content/en/users/data/storage/object-storage/_index.md [hunk 3]
```diff
@@ -342,7 +339,50 @@ to access object storage offered by other providers.
...
+
+$ ./mc cat cesnet/test-egi-2/
```
Comment (sebastian-luna-valero) [inline content/en/users/data/storage/object-storage/_index.md:378]:
```suggestion
$ ./mc cat cesnet/test-egi-public/file1.txt
```

File: content/en/users/data/storage/object-storage/_index.md [hunk 4]
```diff
@@ -354,6 +394,30 @@ $ davix-ls --s3accesskey 'access' --s3secretkey 'secret' \
...
+file-system style operations like cp, mv, ls, du, glob, etc., as well as put/get
+of local files to/from S3.
+
+```python
+import s3fs
+fs = s3fs.S3FileSystem(anon=True,
+      client_kwargs={
+         'endpoint_url': 'https://object-store.cloud.muni.cz'
+      })
+```
```
Comment (sebastian-luna-valero) [inline content/en/users/data/storage/object-storage/_index.md:419]:
```suggestion
```python
from minio import Minio
from minio.error import S3Error

def main():
    client = Minio("object-store.cloud.muni.cz")

    buckets = client.list_buckets()
    for bucket in buckets:
        print(bucket.name, bucket.creation_date)

    # List all object paths in bucket
    objects = client.list_objects('test-egi-public', recursive=True)
    for obj in objects:
        print(obj.bucket_name, obj.object_name.encode('utf-8'), obj.last_modified,
              obj.etag, obj.size, obj.content_type)

    # Get a full object
    data = client.get_object('test-egi-public', 'file1.txt')
    with open('my-testfile', 'wb') as file_data:
        for d in data.stream(32*1024):
            file_data.write(d)

if __name__ == "__main__":
    try:
        main()
    except S3Error as exc:
        print("error occurred.", exc)
```
```

File: content/en/users/data/storage/object-storage/_index.md [hunk 5]
```diff
@@ -354,6 +394,30 @@ $ davix-ls --s3accesskey 'access' --s3secretkey 'secret' \
...
+
```
Comment (sebastian-luna-valero) [inline content/en/users/data/storage/object-storage/_index.md:420]:
```suggestion

There is a good collection of examples on the
[S3Fs GitHub repository](https://github.com/minio/minio-py/tree/master/examples).
```

**Review by sebastian-luna-valero — APPROVED** [review]
Thanks! I have updated the commands with the new test bucket.

File: content/en/users/data/storage/object-storage/_index.md (hunk 4 above)
Comment (sebastian-luna-valero) [reply content/en/users/data/storage/object-storage/_index.md:419]: arg, example for the wrong library! :D 
let me resubmit this change

File: content/en/users/data/storage/object-storage/_index.md [hunk 6]
```diff
@@ -354,6 +394,38 @@ $ davix-ls --s3accesskey 'access' --s3secretkey 'secret' \
...
+      client_kwargs={
+         'endpoint_url': 'https://object-store.cloud.muni.cz'
+      })
+
+print(fs.ls('s3://test-egi-public'))
+s3path = 's3://test-egi-public/file1.txt'
+
+with fs.open(s3path, 'rb') as f:
+    print(f.read())
+
```
Comment (andrea-manzi) [inline content/en/users/data/storage/object-storage/_index.md:426]:
```suggestion
```

```

**Comment by sebastian-luna-valero** [comment]
I would like to add [Rclone](https://rclone.org/) to the list (also available in [conda-forge](https://anaconda.org/conda-forge/rclone)!)

I struggled to add it as a suggestion, so here is the commit for your reference: https://github.com/EGI-Federation/documentation/pull/493/commits/bcd3f8ce8474d9ce5084f4b6cd12ab6880619ced

**Review by gwarf — APPROVED** [review]
LGTM

**Comment by andrea-manzi** [comment]
@gwarf thanks a lot!

**Pull request merged**